packages/neuvector/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,32 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,10 +1,28 @@
++annotations:
++  catalog.cattle.io/auto-install: neuvector-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: NeuVector
++  catalog.cattle.io/kube-version: '>=1.18.0-0'
```

Comment:
Does neuvector support k8s 1.22? if it doesn't make sure to set this to `catalog.cattle.io/kube-version: '>=1.18.0-0 < 1.22.0-0`

Reply:
Neuvector supports 1.22 as well 1.23.